paths/guadosalam.py

```diff
@@ -1,5 +1,49 @@
 from paths.base import AreaMovementBase
 
+class GuadoStart(AreaMovementBase):
+    checkpoint_coordiantes = {
+        0: [-4, -5],
+        1: [-21, 40],
+        2: [-21, 83],
+        3: [-7, 178],
+        4: [0, 0], # Into the door
```

Comment:
I really like the comment here. They really help us see what each check-point is doing.

Reply:
Thank you! I've put in a hand full more.